environment.yml

```diff
@@ -5,7 +5,7 @@ channels:
 dependencies:
   - python>=3.6
   - numpy==1.14
-  - sympy==1.1
+  - sympy==1.2
```

Comment:
I'm merging this, but don't we rather want >= ? not sure why it's still == in the first place